eahub/templates/account/password_change.html

```diff
@@ -5,9 +5,16 @@
 {% block content %}
 <div class="container user-action">
 <h1>Change Password</h1>
-<form method="POST" action="{% url 'account_change_password' %}" class="password_change">
+<form method="POST" action="{% url 'account_change_password' %}">
       {% csrf_token %}
-      {{ form.as_p }}
+      {% for field in form %}
```

Comment:
There's no need to break up the form. You can use `mark_safe` in `settings.py` like in the Stack Overflow answer.

Reply:
Implemented.